### ISSUE TITLE
On Epic, "Launch in VR" from Epic UI causes "[21:33:03 ERR] No selected project"

### ISSUE THREAD

**Issue by code-monet** [rfvgyhn/min-ed-launcher#123]
**Describe the bug/error**

I am trying to use mini ED launcher with Epic without any launch parameters (I want to select the product to launch, and whether to play in VR). However, launching this way does not allow switching to HMD from the in-game Graphics options - it's not an option.

If I then use the Epic Launcher's "Launch in VR" option, I get this error: "[21:33:03 ERR] No selected project"

The only way I can get it to work is by passing `/vr /edo` but of course that takes away the option to select the product to launch.

(If Epic had an option to save shortcuts with different launch parameters, like Steam does, I could work around this easily).

**Debug log**

This seems to be the relevant bit:

```
2024-04-01 21:33:03.296 -07:00 [INF] Available Products:
	Elite Dangerous: Horizons FORC-FDEV-DO-38-IN-40 Up to Date
	Legacy Horizons           FORC-FDEV-D-1013      Up to Date
	Legacy Elite Dangerous    FORC-FDEV-D-1010      Up to Date
	Elite Dangerous: Arena    FORC-FDEV-D-1012      Up to Date
2024-04-01 21:33:03.317 -07:00 [ERR] No selected project
```

**Comment by rfvgyhn** [comment]
Can you provide the log entries up to the `[INF] Elite Runtime` entry?

**Comment by code-monet** [comment]
[min-ed-launcher.log](https://github.com/rfvgyhn/min-ed-launcher/files/14844940/min-ed-launcher.log)

Created a new log file (and removed some names and such)

**Comment by rfvgyhn** [comment]
Ah right, auto run is set to true because `/vr` was specified. This is how the default launcher works (or at least used to when I first wrote the arg parsing) and so I copied that logic.

I think the only workaround until I can fix this is to use legendary/heroic and create individual shortcuts with them.

**Comment by rfvgyhn** [comment]
You can use the latest [CI build](https://github.com/rfvgyhn/min-ed-launcher/actions/runs/8531602785) until the next release if you don't want to use legendary/heroic.

**Comment by code-monet** [comment]
Thanks! I tried it out and this bug is gone. However it keeps trying to install "single player combat demo" (the only available product not installed). If I deleted that folder from products it tries to install it again. This is not happening on reverting to the original Frontier launcher.

I'm not sure if this is a related bug or something separate or something specific to my older config files - let me know if you want to debug more (perhaps in a separate issue).
